cql3/functions/user_aggregate.hh

```diff
@@ -33,7 +33,7 @@ public:
     const scalar_function& sfunc() const {
         return *_sfunc;
     }
-    const scalar_function& reducefunc() const {
+    scalar_function& reducefunc() const {
```

Comment:
Seems wrong. The function itself should not be mutable.

Reply:
Fixed